<launch>

  <node name="myocto" pkg="orb_slam2_ros"
      type="orb_slam2_ros_myocto" output="screen">
       <!--<remap from="/camera/image_raw" to="/mynteye/left_rect/image_rect" />-->

       <!-- static parameters -->
       <param name="all_kfs_pts_topic" type="string" value="/orb_slam2_mono/all_kfs_pts" />
       <param name="single_kf_pts_topic" type="string" value="/orb_slam2_mono/single_kf_pts" />
       <param name="frame_id" type="string" value="/map" />
       <param name="resolution" type="double" value="0.05" />
       <param name="z_min" type="double" value="-0.1" />
       <param name="z_max" type="double" value="0.8" />
  </node>

</launch>
